src/test_l1_dsVideoPort.c

```diff
@@ -171,7 +171,11 @@ void test_l1_dsVideoPort_negative_dsVideoPortInit(void) {
 
 	// Step 02: Attempt to initialize again without terminating the first initialization
 	status = dsVideoPortInit();
-	DS_ASSERT_AUTO_TERM_NUMERICAL(status, dsERR_ALREADY_INITIALIZED);
+	#ifdef ENABLE_ENHANCED_ERROR_CODE
```

Comment:
https://github.com/rdkcentral/ut-core/discussions/55 -> please update to support the KVP module , so we can commit this code as a valid upgrade, that can change configuration at runtime.

Reply:
Done